sass/layout/_footer.scss

```diff
@@ -0,0 +1,47 @@
+footer {
+  @extend %background-accent;
+  position: relative;
+
+  .to-top {
+    position: absolute;
+    text-align: left;
+    margin-left: 2%;
+    top: -1.8em;
+  }
+
+  p {
+    text-align: center;
+    font-family: $primary-font;
+    font-weight: $font-weight-l;
+    font-size: $font-smaller;
+    margin: 0;
+    padding: 2em;
+  }
+
+  .logo {
+    font-family: $logo-font;
+    font-size: em(26);
+    padding: 0;
+    @include logo-underline(7.1%, url('/img/logo-path-white.svg'));
+  }
+
+  ul {
+    padding: 2% 0;
+    margin-bottom: 0;
+    text-align: center;
+
+    li {
```

Comment:
Generally, `li`s that are already scoped within a component do not also need to be nested under a `ul`. Remember the selector chain that Sass will generate. Stay shallow.